dbsync/common/src/main/java/com/google/edwmigration/dbsync/common/InstructionReceiver.java

```diff
@@ -69,4 +64,19 @@ public void close() throws IOException {
       out.close();
     }
   }
-}
+
+  private void flushCopy() throws IOException {
+    if (copyStart == -1) {
+      return;
+    }
+    if (DEBUG) {
+      logger.info(String.format("Reuse bytes from %d for %d bytes", copyStart, copyLength));
+    }
+    in.slice(copyStart, copyLength).copyTo(out);
+
+    // Clear pending copy.
+    copyStart = -1;
+    copyLength = 0;
+  }
```

Comment:
Nit: Could we put this function back "where it was" to minimize the size of this diff?